src/Test.UnitTests.Sarif.Driver/Sdk/AnalyzeCommandBaseTests.cs

```diff
@@ -36,6 +36,44 @@ public AnalyzeCommandBaseTests(ITestOutputHelper output)
             Output.WriteLine($"The seed that will be used is: {TestRule.s_seed}");
         }
 
+        [Fact]
+        public void AnalyzeCommandBase_ScanWithNoValidTargetsAndOneOrMoreSkippedFilesEmitsSkippedFilesWarning()
+        {
+            var logger = new TestMessageLogger();
+
+            var artifact = new EnumeratedArtifact(new FileSystem())
+            { 
+                Uri = new Uri(@"c:\testfile.txt"),
+                Contents = new string('x', 1025), // Just exceeds 1k.
+            };
+
+            var context = new TestAnalysisContext
+            {
+                TargetsProvider = new ArtifactProvider(new[] { artifact }),
+                MaxFileSizeInKilobytes = 1,
+                Logger = logger,
+            };
+
+            int result = new TestMultithreadedAnalyzeCommand().Run(options: null, ref context);
+
+            RuntimeConditions conditions = RuntimeConditions.NoValidAnalysisTargets |
+                                           RuntimeConditions.OneOrMoreFilesSkippedDueToExceedingSizeLimits;
+
+            context.RuntimeErrors.Should().Be(conditions);
+
+            logger.ConfigurationNotifications.Where(n => n.Level == FailureLevel.Note).Count().Should().Be(1);
+            Notification note = logger.ConfigurationNotifications.Where(n => n.Level == FailureLevel.Note).First();
+            note.Descriptor.Id.Should().Be(Notes.Msg002_FileExceedingSizeLimitSkipped);
+
+            logger.ConfigurationNotifications.Where(n => n.Level == FailureLevel.Warning).Count().Should().Be(1);
+            Notification warning = logger.ConfigurationNotifications.Where(n => n.Level == FailureLevel.Warning).First();
+            warning.Descriptor.Id.Should().Be(Warnings.Wrn997_OneOrMoreFilesSkippedDueToExceedingSizeLimits);
+
+            logger.ConfigurationNotifications.Where(n => n.Level == FailureLevel.Note).Count().Should().Be(1);
```

Comment:
>FailureLevel.Note [](http://example.com/codeflow?start=68&length=17)

I think the purpose of this change is, 
the visibility of file skipped is low to user (for user that has Note).
It is ok (I think not too bad), 
but when all files are oversized, 
people will appreciate more visibility of it.

In other tools that generates SARIF file, 
we generates warning and promote to error in some condition.
How about we do similar promote the visibility from note to warning, 
instead of producing another message? 

The information included in the note has all and more.

Reply:
I found whether 
`all files are too big`
or
`at least one is within limit`
we will always output this message, so to understand, this effectively can be shorten to understand as 
"always report a warning, if at least one file skipped"

if this behavior is what we want, then we can remove the other copy of  `if (_filesExceedingSizeLimitCount > 0){Warnings.LogOneOrMoreFilesSkippedDueToExceedingSizeLimit(globalContext, filesExceedingSizeLimitCount); }`
and move your copy outside of the check `if (_fileContextsCount == 0)`
for less copy of same code